packages/vm/src/calls.rs

```diff
@@ -106,7 +104,7 @@ where
     A: BackendApi + 'static,
     S: Storage + 'static,
     Q: Querier + 'static,
-    U: DeserializeOwned + Clone + fmt::Debug + JsonSchema + PartialEq,
+    U: DeserializeOwned + CustomMsg,
```

Comment:
Seems like `DeserializeOwned` should be a requirement for `CustomMsg`

Reply:
We only need to Serialize it in wasm.
Makes sense to add it.

Reply:
Ahh, so this is only needed for integration tests where the deserialization happens in the host. In this case it is not strictly needed. However, it would be nice if contract developers don't try to be claver and build a crazy reference mess.

Reply:
> However, it would be nice if contract developers don't try to be claver and build a crazy reference mess.

contract developers do not implement CustomMsg.
This is done once by Terra, once by CyberCongress. Many blockchains will never implement one.
Contract developers will just import one.

Reply:
Ah, right